editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -3573,30 +3573,30 @@ void TilesetEditorContext::_get_property_list(List<PropertyInfo> *p_list) const
-			if (tileset->tile_get_tile_mode(id) == TileSet::AUTO_TILE) {
-				p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_autotile_bitmask_mode"), PROPERTY_HINT_ENUM, "2x2,3x3 (minimal),3x3"));
-				p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_subtile_size")));
-				p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_subtile_spacing"), PROPERTY_HINT_RANGE, "0, 1024, 1"));
-			} else if (tileset->tile_get_tile_mode(id) == TileSet::ATLAS_TILE) {
-				p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_subtile_size")));
-				p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_subtile_spacing"), PROPERTY_HINT_RANGE, "0, 1024, 1"));
-			}
-			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_occluder_offset")));
-			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_navigation_offset")));
-			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_shape_offset"), PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR));
-			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_shape_transform"), PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR));
-			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_z_index"), PROPERTY_HINT_RANGE, itos(VS::CANVAS_ITEM_Z_MIN) + "," + itos(VS::CANVAS_ITEM_Z_MAX) + ",1"));
-		}
+	}
+	if (tileset_editor->get_current_tile() >= 0 && !tileset.is_null()) {
+		int id = tileset_editor->get_current_tile();
+		p_list->push_back(PropertyInfo(Variant::NIL, GNAME("Selected Tile", "tile_"), PROPERTY_HINT_NONE, "tile_", PROPERTY_USAGE_GROUP));
+		p_list->push_back(PropertyInfo(Variant::STRING, PNAME("tile_name")));
+		p_list->push_back(PropertyInfo(Variant::OBJECT, PNAME("tile_texture"), PROPERTY_HINT_RESOURCE_TYPE, "Texture"));
+		p_list->push_back(PropertyInfo(Variant::OBJECT, PNAME("tile_normal_map"), PROPERTY_HINT_RESOURCE_TYPE, "Texture"));
+		p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_tex_offset")));
+		p_list->push_back(PropertyInfo(Variant::OBJECT, PNAME("tile_material"), PROPERTY_HINT_RESOURCE_TYPE, "ShaderMaterial"));
+		p_list->push_back(PropertyInfo(Variant::COLOR, PNAME("tile_modulate")));
+		p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_tile_mode"), PROPERTY_HINT_ENUM, "SINGLE_TILE,AUTO_TILE,ATLAS_TILE"));
+		if (tileset->tile_get_tile_mode(id) == TileSet::AUTO_TILE) {
+			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_autotile_bitmask_mode"), PROPERTY_HINT_ENUM, "2x2,3x3 (minimal),3x3"));
+			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_subtile_size")));
+			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_subtile_spacing"), PROPERTY_HINT_RANGE, "0, 1024, 1"));
+		} else if (tileset->tile_get_tile_mode(id) == TileSet::ATLAS_TILE) {
+			p_list->push_back(PropertyInfo(Variant::VECTOR2, PNAME("tile_subtile_size")));
+			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_subtile_spacing"), PROPERTY_HINT_RANGE, "0, 1024, 1"));
```

Comment:
Please re-add this line you've removed in #64931 when rebasing from https://github.com/godotengine/godot/commit/29a69cb51e260fb7be3a4d0bd2396c06ea0beb40 to https://github.com/godotengine/godot/commit/211c707e67725f27ae995a923dd206cca9976315. Originally it was added in #71533.
And please be more careful when rebasing / force pushing. :upside_down_face:

```suggestion
			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_subtile_spacing"), PROPERTY_HINT_RANGE, "0, 1024, 1"));
			p_list->push_back(PropertyInfo(Variant::INT, PNAME("tile_autotile_fallback_mode"), PROPERTY_HINT_ENUM, "Auto,Icon"));
```

Reply:
It had been a years-old PR, ow 💦💦💦

Reply:
Sorry but I've suggested it on a wrong line. :laughing: It should be within `if (tileset->tile_get_tile_mode(id) == TileSet::AUTO_TILE)` block.

Reply:
We're all collectively f\*\*\*\*ups